extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/HttpConfiguration.java

```diff
@@ -197,6 +198,18 @@
     @ConfigItem
     public Map<String, SameSiteCookieConfig> sameSiteCookie;
 
+    /**
+     * Configuration that allows setting forwarded host header to be considered.
+     */
+    @ConfigItem(defaultValue = "x_forwarded_host", name = "forwarded-host-header")
```

Comment:
I think you can drop `name` attribute here and below

Reply:
Done!